.contributing/the-process.md

```diff
@@ -96,13 +96,23 @@ Create: 
 
         -   Decide on widget/action/middleware names
 
--   An issue on GitHub for each use case
+-   An issue on GitHub for each use case:
 
-    1.  Add a section label (eg. Auth) 
+    1.  Create a tracking issue
 
-    2.  Indicate the use case it addresses & link to the Google doc  
+        - add the "tracking" label
 
--   Links back and forth between the Task, Issue & Doc 
+        - Add a title "section_name use_case_number: use_case_name"
+
+        - in the description, break down the use case into a checklist of tasks (sub issues)
+
+        - grab team for discussion/ input on checklist
```

Comment:
@nickmeinhold, @tamari-gray Do you guys think the team needs to weigh in on the sub-issue checklist? We can always edit and comment later if need be

Reply:
I don't have strong feelings, I'd probably say no, but we could try either way and change later if we want to

Reply:
If anyone has a strong opinion let's do that I reckon

Reply:
I'm leaning no also, but willing to be convinced otherwise!

Reply:
happy for no aswell!!

Reply:
ill update the pr!